bin/setup

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-
-rm -rf node_modules && npm i --no-save
```

Comment:
In order to keep the lock files in check between different Operating Systems, we will need to include the `--no-save` option